app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -350,12 +352,20 @@ class EditPatientScreen(context: Context, attributeSet: AttributeSet) : Relative
           showDateOfBirthIsInFutureError()
         }
 
-        AGE_INVALID -> {
-          showAgeIsInvalidError()
+        AGE_EXCEEDS_MAX_LIMIT -> {
+          showAgeExceedsMaxLimitError()
```

Comment:
If its just one statement in `when` block, you can get rid of the braces.
Please format this method to remove braces everywhere.